docs/_static/theme_overrides.css

```diff
@@ -0,0 +1,13 @@
+/* override table width restrictions */
```

Comment:
All of the css / renaming value->valu, group->cluster looks good. Starting to review the storm docs now.